src/app/modules/home/components/home-section/homes-section.component.html

```diff
@@ -34,15 +34,11 @@ <h1 class="display-4">Get
             </div>
           </div>
         </div>
-
-        <div class="col-lg-4">
-          <div class="card register-signup text-center card-form">
-            <div class="card-body">
-              <h3>Sign Up Today</h3>
-              <p>Please fill out this form to register</p>
-              <app-register-form></app-register-form>
-            </div>
-          </div>
+        <div class="col text-center py-5">
+          <h1 class="display-4">SignUp</h1>
+          <p class="lead">Go to regiseter page and sign up, or login if you are existing user</p>
```

Comment:
Typo: should be register instead of 'regiseter'.

Reply:
fixed by:
https://github.com/anowakowski/PictureApp_SPA/pull/25/commits/c756011b6cfe7fe12c9a6b15ddaa543cb1ec0117